README.md

```diff
@@ -1,6 +1,6 @@
 # Lamoda metric responder
 
-[![Build Status](https://travis-ci.org/lamoda/php-metrics.svg?branch=master)](https://travis-ci.org/lamoda/php-metrics)
+[![Build Status](https://github.com/lamoda/php-metrics/workflows/PHP%20Composer/badge.svg?branch=master)](https://github.com/lamoda/php-metrics/workflows/PHP%20Composer/badge.svg?branch=master)
```

Comment:
Text on the badge is a but weird. Can we get here something like `build`, `tests` or similar?

Reply:
Yes, we can change name [here](https://github.com/lamoda/php-metrics/blob/master/.github/workflows/php.yml#L1)